priv/repo/migrations/20161028045343_add_files_table.exs

```diff
@@ -5,22 +5,20 @@ defmodule EdmBackend.Repo.Migrations.AddFilesTable do
     create table(:files, primary_key: false) do
       add :id, :uuid, primary_key: true, default: fragment("uuid_generate_v4()")
 
-      add :filepath, :text
-      add :size, :integer
+      add :filepath, :text, null: false
+      add :filepath_md5, :string, size: 32, null: false
```

Comment:
Not very important, but should this just be `md5`/`md5_checksum`/`file_md5` or similar? "filepath" seems redundant.